lib/rules/web/admin_console/4.2/common_ui_elements.xyaml

```diff
@@ -171,7 +171,10 @@ check_table_text_data:
   element:
     selector:
       xpath: //td[text()='<text_data>']
-
+check_table_line_count:
+  scripts:
```

Comment:
it's better to steps to make sure element appears before running scripts, then user can omit the step `wait_table_loaded` step